src/__stories__/carousel-story.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {
+    Carousel,
+    Box,
+    ResponsiveLayout,
+    MediaCard,
+    ButtonLink,
+    Callout,
+    Stack,
+    IconInformationRegular,
+    SectionTitle,
+    Text2,
+} from '..';
+
+export default {
+    title: 'Components/Carousel/Carousel',
+};
+
+type Args = {
+    numItems: number;
+    itemsPerPageMobile: number;
+    itemsPerPageDesktop: number;
+    withBullets: boolean;
+    nextPageOffset: number;
+    prevPageOffset: number;
+    free: boolean;
+    itemsToScroll: number;
+    autoplay: boolean;
+};
+
+export const Default: StoryComponent<Args> = ({
+    numItems,
+    withBullets,
+    itemsPerPageMobile,
+    itemsPerPageDesktop,
+    nextPageOffset,
+    prevPageOffset,
+    free,
+    itemsToScroll,
+    autoplay,
+}) => {
+    const [pageInfo, setPageInfo] = React.useState<{
+        pageIndex: number;
+        shownItemIndexes: Array<number>;
+    } | null>(null);
+    return (
+        <Box paddingY={24}>
+            <ResponsiveLayout>
+                <Stack space={16}>
+                    <Callout
+                        description="Arrow controls disappear in touch devices"
+                        icon={<IconInformationRegular />}
+                    />
+                    <Carousel
+                        withBullets={withBullets}
+                        free={free}
+                        itemsPerPage={{mobile: itemsPerPageMobile, desktop: itemsPerPageDesktop}}
```

Comment:
Yes, I thought about adding that, but it would increase the complexity. Do you think it will be useful? do you think we'll use carousels with different number of items per page in mobile and tablet? I think we don't have that case in any of the implemented carousels in webapp today.

Reply:
We can let this case for a future evolution but I'm afraid about using carousels with cards in tablets that cause huge cards sizes in tablet views. It is something that is happening nowadays using cards without carousel, but..

Maybe we need to add to the mistica roadmap to work to improve Tablet experience

![image](https://user-images.githubusercontent.com/6722153/152322864-5593c632-133e-4bf4-b057-b75407c97e7e.png)